### PR TITLE
clarify RLIMIT_AS is disabled, explain how to edit DOCKER_OPTS 

### PR DESCRIPTION
**Pull request by dalanlan**
fix issue #11676 , #11754 
1 Clarify the RLIMIT_AS for --ulimit is no longer supported.
2 Explain how to edit DOCKER_OPTS for both ubuntu and centos.
Signed-off-by: Simei He hesimei@zju.edu.cn

### CODE REVIEW

**Comment by dalanlan** [comment]
@cpuguy83

File: docs/sources/articles/networking.md [hunk 1]
```diff
@@ -121,8 +121,17 @@ Finally, several networking options can only be provided when calling
  *  `-P` or `--publish-all=true|false` — see
     [Binding container ports](#binding-ports)
 
-The following sections tackle all of the above topics in an order that
-moves roughly from simplest to most complex.
+For those networking options which you would like to supply to the Docker server when it starts up, you can set them up using `DOCKER_OPTS` in the Docker upstart configuration file. For ubuntu, edit the variable in /etc/default/docker while /etc/sysconfig/docker for centos.
```
Comment (moxiegirl) [inline docs/sources/articles/networking.md]: The wrap should be to 80 characters. There are some English construction errors.  For an update see this gist:

https://gist.github.com/moxiegirl/a354e23c0cc9af0cd124

File: docs/sources/reference/commandline/cli.md [hunk 2]
```diff
@@ -2195,6 +2195,8 @@ available in the default container, you can set these using the `--ulimit` flag.
 > If you do not provide a `hard limit`, the `soft limit` will be used for both
 values. If no `ulimits` are set, they will be inherited from the default `ulimits`
 set on the daemon.
+> `RLIMIT_AS` is disabled for now. In other words, the following script is not supported:
```
Comment (cpuguy83) [inline docs/sources/reference/commandline/cli.md]: `RLIMIT_AS` probably doesn't make sense to the end user, and should _probably_ be referred to as `address space` (or `as` as that is the option name).

File: docs/sources/articles/networking.md [hunk 3]
```diff
@@ -121,8 +121,23 @@ Finally, several networking options can only be provided when calling
  *  `-P` or `--publish-all=true|false` — see
     [Binding container ports](#binding-ports)
 
-The following sections tackle all of the above topics in an order that
-moves roughly from simplest to most complex.
+To supply networking options to the Docker server at startup, use the
+`DOCKER_OPTS` in the Docker upstart configuration file. For Ubuntu, edit the
```
Comment (jamtur01) [inline docs/sources/articles/networking.md]: use the \`DOCKER_OPTS\` variable...

File: docs/sources/articles/networking.md [hunk 4]
```diff
@@ -121,8 +121,23 @@ Finally, several networking options can only be provided when calling
...
+variable in `/etc/default/docker` and `/etc/sysconfig/docker` for Centos.
```
Comment (jamtur01) [inline docs/sources/articles/networking.md]: or for Centos, \`/etc/...\`

**Comment by jamtur01** [comment]
LGTM minor edits. Ping @moxiegirl @SvenDowideit @fredlf

File: docs/sources/reference/commandline/cli.md [hunk 5]
```diff
@@ -2195,6 +2195,8 @@ available in the default container, you can set these using the `--ulimit` flag.
 > If you do not provide a `hard limit`, the `soft limit` will be used for both
 values. If no `ulimits` are set, they will be inherited from the default `ulimits`
 set on the daemon.
+> `as` option is disabled now. In other words, the following script is not supported:
+>   `$docker run -it --ulimit as=1024 fedora /bin/bash`
```
Comment (jfrazelle) [inline docs/sources/reference/commandline/cli.md]: there should be a space between the $ and docker

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**